launcher/ui/themes/DarkTheme.cpp

```diff
@@ -18,18 +18,18 @@ bool DarkTheme::hasColorScheme()
 QPalette DarkTheme::colorScheme()
 {
     QPalette darkPalette;
-    darkPalette.setColor(QPalette::Window, QColor(49,54,59));
+    darkPalette.setColor(QPalette::Window, QColor(49,49,49));
     darkPalette.setColor(QPalette::WindowText, Qt::white);
-    darkPalette.setColor(QPalette::Base, QColor(35,38,41));
-    darkPalette.setColor(QPalette::AlternateBase, QColor(49,54,59));
+    darkPalette.setColor(QPalette::Base, QColor(34,34,34));
+    darkPalette.setColor(QPalette::AlternateBase, QColor(38,38,38));
```

Comment:
similar to the bright theme, this alternating color is too close to the base color imo
before:
![image](https://user-images.githubusercontent.com/9145768/193423177-68baf771-e0d5-4fb1-9f34-4d10904859fb.png)
after:
![image](https://user-images.githubusercontent.com/9145768/193423210-5731d41e-1a81-4431-9945-11773b62b45c.png)